### PR TITLE
Multiqc@0.1.0

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

**Review by abenjak — CHANGES_REQUESTED** [review]
`Dockerfile`: replace `latest` with the actual version

The rest looks good :)

**Comment by lindaxiang** [comment]
> `Dockerfile`: replace `latest` with the actual version
> 
> The rest looks good :)

Thanks @abenjak . Very good point. :-)

**Review by abenjak — APPROVED** [review]

File: multiqc/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Package multiqc
+
+
+Please update this with a brief introduction of the package.
```
Comment (pruzanov) [inline multiqc/README.md:4]: This is a placeholder, is this intentional? Is there going to be some actual documentation here?

**Review by pruzanov — APPROVED** [review]

File: multiqc/README.md (hunk 1 above)
Comment (lindaxiang) [reply multiqc/README.md:4]: Thanks @pruzanov . Glad you point this out and indeed we need to have good documentation. I will work on this.

Comment (lindaxiang) [reply multiqc/README.md]: FYI @pruzanov,  I've updated the `README.md`.

**Pull request merged**